MantidPlot/CMakeLists.txt

```diff
@@ -910,7 +910,7 @@ else ()
 endif ()
 
 # Add an environment property MANTID_SOURCE so that the script can find the source of xmlrunner
-set (TEST_ENVIRONMENT "MANTID_SOURCE=${CMAKE_SOURCE_DIR};PYTHONPATH=${PYTHON_XMLRUNNER_DIR};MANTID_SCREENSHOT_REPORT=${SCREENSHOTS_DIR}" )
+set (TEST_ENVIRONMENT "MANTID_SOURCE=${CMAKE_SOURCE_DIR};PYTHONPATH=$ENV{PYTHONPATH}:${PYTHON_XMLRUNNER_DIR};MANTID_SCREENSHOT_REPORT=${SCREENSHOTS_DIR}" )
```

Comment:
So this will only capture the environment variable at cmake configure time and not at ctest time.  think that's the best we can do though?

Reply:
Yes, I think that's exactly what we need here to capture at config time. This fixes the problem on my server.

Reply:
I just wanted to double check. Thanks.